orchestrator/src/server.ts

```diff
@@ -30,6 +31,11 @@ import definitionsJSON from "./schemas/definitions.json";
 import { docker, getPreprocessorServices, getHandlerServices } from "./docker";
 
 const app = express();
+const memjsClient = Client.create();
```

Comment:
I don't like caching server selection being purely automatic. I can imagine users wanting to be able to have multiple servers running for different purposes, and having issues with how this is currently configured.